code/modules/mob/mob_helpers.dm

```diff
@@ -47,11 +47,11 @@
 	return istype(M) && M.player_logged && M.stat != DEAD
 
 /proc/isAntag(A)
-	if(istype(A, /mob/living/carbon))
-		var/mob/living/carbon/C = A
-		if(C.mind && C.mind.special_role)
-			return 1
-	return 0
+	if(isliving(A))
+		var/mob/living/L = A
+		if(L.mind?.special_role)
+			return TRUE
```

Comment:
```suggestion
	var/mob/living/L = A
	if(L?.mind?.special_role)
		return TRUE
			
```

Reply:
Removing the check would cause runtimes if `isAntag()` is run on an object.
`L?.` doesn't replace that since it only checks that `L` isn't `null`.

Reply:
My bad - I was severely misinformed!
Thanks for the digilence!